_studio/shared/umc/codec/h264_dec/src/umc_h264_task_supplier.cpp

```diff
@@ -3610,6 +3610,8 @@ Status TaskSupplier::AddSlice(H264Slice * pSlice, bool force)
             {
                 SetOfSlices * setOfSlices = m_accessUnit.GetLayer(i);
                 H264Slice * slice = setOfSlices->GetSlice(0);
```

Comment:
this variable looks to be loop invariant. Maybe we should skip entire loop if it is null?

Reply:
slice - different for each iteration: comes from setOfSlices for each layer (setOfSlices = m_accessUnit.GetLayer(**i**);)

Reply:
you are right, I missed it